### ISSUE TITLE
Number of Page views different in table view (Pages) and transitions popup

### ISSUE THREAD

**Issue by HTWGkonstanz** [piwik/piwik#9024]
Pageviews totally differ from the "standard" table view to the "transitions" view regardless which date range is picked or wether you look at pages or page titles.

Sometime the number in the "transisionts" view is the higher one, most of the time the number in den "table" view is the higher one, some numbers are the same. Some numbers only differ by 1-2, some numbers differ by 1.000 - it doesn´t make any sense.

I thought, they should be the same...what aspect am I missing here? 

![r669dvhufa](https://cloud.githubusercontent.com/assets/15153738/10536540/2872d34a-73ec-11e5-82ac-90b4666b640f.jpg)

![tmvck9y4mc](https://cloud.githubusercontent.com/assets/15153738/10536541/2873f6c6-73ec-11e5-9f0a-1597563cf981.jpg)

**Comment by mattab** [comment]
Hi @HTWGkonstanz 

Thanks for the report, this is surprising indeed: it should be the same, because both views should use the same API and data source.

Can you reproduce this issue on https://demo.piwik.org ? please paste the links and we will investigate

**Comment by HTWGkonstanz** [comment]
Dear Matthieu Aubry,

 

yes, I can replicate the bug – but the differences between the numbers are much smaller (differences of 10 views etc.)

 

For example:
Pages
In year 2015 the page „index“ has 36.203 page views.

If you open “Transitions” for this pages, it shows 36.185 page views.


https://demo.piwik.org/index.php?module=CoreHome <https://demo.piwik.org/index.php?module=CoreHome&action=index&idSite=7&period=year&date=2015-11-26#?module=Actions&action=menuGetPageUrls&idSite=7&period=year&date=2015-11-26> &action=index&idSite=7&period=year&date=2015-11-26#?module=Actions&action=menuGetPageUrls&idSite=7&period=year&date=2015-11-26

 

 

On our website we had much more “extreme” differences, where one number was only 50% of the other number.

 


Kind regards,

Julia Zádor

 

--

Kommunikation


HOCHSCHULE KONSTANZ

Technik, Wirtschaft und Gestaltung (HTWG) 
Raum A323

Brauneggerstr. 55 | D-78462 Konstanz


Fon:    +49 7531 206-636

Fax:    +49 7531 206-87636

E-Mail: julia.zador@htwg-konstanz.de

Web:    http://www.htwg-konstanz.de <http://www.htwg-konstanz.de/> 

 

Die HTWG bei Facebook: http://www.facebook.com/HTWGkonstanz

Das Portal zum Studium am See: http://www.grenzenlos-studieren.de <http://www.grenzenlos-studieren.de/> 

 

Von: Matthieu Aubry [mailto:notifications@github.com] 
Gesendet: Donnerstag, 26. November 2015 02:52
An: piwik/piwik
Cc: HTWGkonstanz
Betreff: Re: [piwik] Number of Page views different in table view (Pages) and transitions popup (#9024)

 

Hi @HTWGkonstanz <https://github.com/HTWGkonstanz>  

Thanks for the report, this is surprising indeed: it should be the same, because both views should use the same API and data source.

Can you reproduce this issue on https://demo.piwik.org ? please paste the links and we will investigate

**Comment by sgiehl** [comment]
Guess we need to have a closer look at the way the transitions are calculating their data.
It seems to be calculated on the given URL. Which might not always be correct.

Btw. it is also possible to display data for invalid URLs. I guess there shouldn't be data at all, but instead some strange data is displayed.... See https://demo.piwik.org/index.php?module=CoreHome&action=index&idSite=7&period=year&date=2015-11-26#?module=Actions&action=menuGetPageUrls&idSite=7&period=year&date=2015-11-26&popover=RowAction$3ATransitions$3Aurl$3A$2FinvalidUr12345

**Comment by mgonera** [comment]
Is anyone caring about this in 2017? :) I confirmed that current 3.0 demo on piwik.org also has this issue. It's enough to run transitions on monthly data "index" page.

**Comment by sebastianpiskorski** [comment]
Yes, I've checked this one on large dataset and discrepancies still occur.

**Comment by rogogon** [comment]
hi ;) I want to confirm that this bug is still occurring. Are you planning to fix it?

**Comment by sgiehl** [comment]
@rogogon It's currently not planned to work on this issue. A Pull Request would be very welcome. Or if you would like to sponsor this fix/change feel free to [contact the Piwik team](https://piwik.org/contact/). Otherwise it might take some time.

**Comment by atom-box** [comment]
I think I am seeing this problem but it is inconsistent. In testing, on a single website I could only reproduce it in one of five cases.

![Image](https://github.com/user-attachments/assets/5902acab-3c56-45b3-8064-202cfb47ae33)

**Bug:**

In Behavior >> Transitions: The top of the card shows 111,483 pageviews.

In Behavior >> Pages: 12,538 pageviews



**Version of Matomo:** Matomo Cloud (5.3.0)



**To reproduce:**

Open the same time period and the same page in Transitions and in Pages.

Notice the total pageviews is different



**Testing results:**
 so far I found it in one of five situations. See screenshots, see the part I circled in red.
